packages/cactus-cmd-api-server/Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM node:12.20.1-alpine3.12
+
+ARG NPM_PKG_VERSION=latest
+
+RUN apk update
+RUN apk add --no-cache tini
+
+ARG APP=/usr/src/app/
+
+ENV TZ=Etc/UTC
+ENV APP_USER=appuser
+
+RUN addgroup --system $APP_USER
+RUN adduser --system $APP_USER -G $APP_USER
+RUN mkdir -p ${APP}
+
+RUN mkdir -p "${APP}/log/"
+RUN chown -R $APP_USER:$APP_USER "${APP}/log/"
+
+WORKDIR ${APP}
+
+
+RUN chown -R $APP_USER:$APP_USER ${APP}
+
+ENV CACTUS_NODE_ID=-
+ENV CONSORTIUM_ID=-
+ENV KEY_PAIR_PEM=-
+ENV COCKPIT_WWW_ROOT=${APP}node_modules/@hyperledger/cactus-cockpit/www/
+ENV COCKPIT_TLS_ENABLED=false
+ENV COCKPIT_CORS_DOMAIN_CSV=\*
+ENV COCKPIT_MTLS_ENABLED=false
+ENV COCKPIT_TLS_CERT_PEM=-
+ENV COCKPIT_TLS_KEY_PEM=-
+ENV COCKPIT_TLS_CLIENT_CA_PEM=-
+ENV COCKPIT_HOST=0.0.0.0
+ENV COCKPIT_PORT=3000
+ENV API_MTLS_ENABLED=false
+ENV API_TLS_ENABLED=false
+ENV API_CORS_DOMAIN_CSV=\*
+ENV API_TLS_CERT_PEM=-
+ENV API_TLS_CLIENT_CA_PEM=-
+ENV API_TLS_KEY_PEM=-
+ENV API_HOST=0.0.0.0
+ENV API_PORT=4000
+ENV LOG_LEVEL=TRACE
+
+# GUI: 3000, API: 4000
+EXPOSE 3000 4000
+
+USER $APP_USER
+
+RUN npm i @hyperledger/cactus-cmd-api-server@${NPM_PKG_VERSION} --production
+
+ENTRYPOINT ["/sbin/tini", "--"]
+CMD ["node", "node_modules/@hyperledger/cactus-cmd-api-server/dist/lib/main/typescript/cmd/cactus-api.js", "filler-until-bug-794-is-fixed"]
```

Comment:
@jordigiam In the meantime #794 (#795) got merged so we should no longer need the workaround dummy parameter here. If you could remove that (after having rebased onto main) and then re-test I'd be grateful. (it can be in the context/scope of any other one of the container PRs I just happened to open this one first)

Reply:
@jordigiam I forgot that the API server image I put on my branch was not yet published so you couldn't use it as a base image just yet. I published it under this tag for now (later will create it under hyperledger as well but that takes more time)
`petermetz/cactus-cmd-api-server:2021-04-18-fix-quorum-contract-types` 

https://hub.docker.com/layers/146080676/petermetz/cactus-cmd-api-server/2021-04-18-fix-quorum-contract-types/images/sha256-78185bdf50281d875215807394bb22cbb12d03c423c8e7c578a9c8fb48f9a209?context=explore

Reply:
@petermetz I'm facing this issue when I try to use your image: "npm ERR! Invalid package name "@petermetz/cactus-cmd-api-server:2021-04-18-fix-quorum-contract-type": name can only contain URL-friendly characters"

Reply:
@jordigiam `petermetz/cactus-cmd-api-server:2021-04-18-fix-quorum-contract-type` is the docker image tag, not an npm package name. For the npm package you'd want this: https://www.npmjs.com/package/@hyperledger/cactus-plugin-ledger-connector-corda/v/0.4.2-main.b067fdd.29

Also now that I think about it, the container image for corda will be a little different than the others because the corda connector runs in the JVM so for this one you'll need some additional customization to the cmd-api-server base image so that it has the JDK installed. Basically you'll want a combination of the cmd-api-server base image and the standalone corda connector image here: `packages/cactus-plugin-ledger-connector-corda/src/main-server/Dockerfile`

Reply:
@jordigiam Other thing I just remembered in addition to the comment above: You'll need to swap out tini for supervisord to be able to manage multiple processes in the container.

Reply:
> @jordigiam I forgot that the API server image I put on my branch was not yet published so you couldn't use it as a base image just yet. I published it under this tag for now (later will create it under hyperledger as well but that takes more time)
> `petermetz/cactus-cmd-api-server:2021-04-18-fix-quorum-contract-types`
> 
> https://hub.docker.com/layers/146080676/petermetz/cactus-cmd-api-server/2021-04-18-fix-quorum-contract-types/images/sha256-78185bdf50281d875215807394bb22cbb12d03c423c8e7c578a9c8fb48f9a209?context=explore

@petermetz That image is not available now, can I use image generated by cactus-cmd-cpi-server Dockerfile?